### PR TITLE
Feature/add extra gateway dp

### PR DESCRIPTION
**Pull request by stevemul**
Add extra floating IP for use as an additional dataplane on the extra gateway network (usually a private network like a VRF). Also adds the routers in openshift that are required to route the traffic in the cluster.

### CODE REVIEW

**Comment by stevemul** [comment]
Tests passed

File: roles/openshiftpostdeployment/tasks/main.yml [hunk 1]
```diff
@@ -78,6 +78,27 @@
...
+- name: Setup private network router environment
+  command: /usr/local/bin/oc set env dc/router-private {{ item }} -n default
+  with_items:
+    - ROUTER_SERVICE_HTTP_PORT=7080
+    - ROUTER_SERVICE_HTTPS_PORT=7443
+    - ROUTE_LABELS="router-private=true"
+  when: extra_gateway_vip is defined
+
+- name: Scale up private network routers
+  command: /usr/local/bin/oc scale dc router-private --replicas=2 -n default
```
Comment (k8soneill) [inline roles/openshiftpostdeployment/tasks/main.yml]: --replicas=2 needs to be changed to match the amount of infra nodes

File: roles/openshiftpostdeployment/tasks/main.yml [hunk 2]
```diff
@@ -78,6 +78,27 @@
...
+  when: extra_gateway_vip is defined
+
+- name: label nodes for private network routers
+  command: /usr/local/bin/oc label node {{ item }}  'router-private=true'
```
Comment (k8soneill) [inline roles/openshiftpostdeployment/tasks/main.yml]: Add this to labels in the host file to ensure they are persistent on reboot

**Review by k8soneill — DISMISSED** [review]
Changes requested

**Review by k8soneill — APPROVED** [review]
Changes have been implemented and tested and proven to work

**Pull request merged**